tests/e2e/virt_backup_restore_suite_test.go

```diff
@@ -0,0 +1,44 @@
+package e2e_test
+
+import (
+	"os"
+	"strings"
+	"time"
+
+	. "github.com/onsi/ginkgo/v2"
+	. "github.com/onsi/gomega"
+	. "github.com/openshift/oadp-operator/tests/e2e/lib"
+)
+
+var virtTestingEnabled bool
+
+func init() {
+	if strings.ToLower(os.Getenv("TEST_VIRT")) == "true" {
+		virtTestingEnabled = true
+	}
+}
+
+var _ = Describe("VM backup and restore tests", Ordered, func() {
+	var v *VirtOperator
+	var err error
+
+	var _ = BeforeAll(func() {
```

Comment:
should we also uninstall it?

Reply:
I tried it this way first, but it got messy dealing with uninstalls stuck on finalizers and trying to only uninstall the operator if the test was the one to install it. It is possible, but I'm not sure it's worth the effort yet.

Reply:
can we just add in the docs how to unistall it then? so someone running locally the test can return the cluster to original state after running the tests

Reply:
Sure, I will see what I can do that is convenient for local tests.